tests/tests/authority_tests.cpp

```diff
@@ -1858,43 +1879,129 @@ BOOST_FIXTURE_TEST_CASE( parent_owner_test, database_fixture )
    }
 }
 
-BOOST_FIXTURE_TEST_CASE( owner_delegation_test, database_fixture )
-{ try {
-   ACTORS( (alice)(bob) );
-
-   fc::ecc::private_key bob_active_key = fc::ecc::private_key::regenerate(fc::digest("bob_active"));
-   fc::ecc::private_key bob_owner_key  = fc::ecc::private_key::regenerate(fc::digest("bob_owner"));
-
-   trx.clear();
+BOOST_AUTO_TEST_CASE( custom_operation_required_auths_before_fork ) {
+   try {
+      ACTORS((alice)(bob));
+      fund(alice, asset(10000000));
+      enable_fees();
 
-   // Make sure Bob has different keys
-   account_update_operation auo;
-   auo.account = bob_id;
-   auo.active = authority( 1, public_key_type(bob_active_key.get_public_key()), 1 );
-   auo.owner  = authority( 1, public_key_type(bob_owner_key.get_public_key()), 1 );
-   trx.operations.push_back( auo );
-   sign( trx, bob_private_key );
-   PUSH_TX( db, trx );
-   trx.clear();
+      if (db.head_block_time() >= HARDFORK_CORE_210_TIME) {
+          wlog("Unable to test custom_operation required auths before fork: hardfork already passed");
```

Comment:
I think we SHOULD test it to avoiding breaking consensus. Nobody notices warnings when batch-running test cases. That said, test with `BOOST_REQUIRE( db.head_block_time() < HARDFORK_CORE_210_TIME )`.